app/src/main/java/org/simple/clinic/settings/changelanguage/ChangeLanguageListItem.kt

```diff
@@ -11,7 +11,7 @@ import org.simple.clinic.widgets.recyclerview.ViewHolderX
 
 data class ChangeLanguageListItem(
     private val language: ProvidedLanguage,
-    private val isSelected: Boolean
```

Comment:
This was renamed because it was actually picking up the `View#isSelected` property instead of the one from the list item.